internal/xds/translator/testdata/out/xds-ir/authn-multi-route-multi-provider.routes.yaml

```diff
@@ -0,0 +1,84 @@
+- name: first-listener
+  virtualHosts:
+  - domains:
+    - '*'
+    name: first-listener
+    routes:
+    - match:
+        path: foo/bar
+      route:
+        cluster: first-route
+      typedPerFilterConfig:
+        envoy.filters.http.jwt_authn:
+          '@type': type.googleapis.com/envoy.extensions.filters.http.jwt_authn.v3.JwtAuthentication
+          providers:
+            example:
+              audiences:
+              - foo.com
+              issuer: https://www.example.com
+              payloadInMetadata: https://www.example.com
+              remoteJwks:
+                cacheDuration: 300s
+                httpUri:
+                  cluster: "127_0_0_1_80"
+                  timeout: 5s
+                  uri: http://127.0.0.1/jwt/public-key/jwks.json
+            example2:
+              audiences:
+              - one.foo.com
+              - two.foo.com
+              issuer: https://www.two.example.com
+              payloadInMetadata: https://www.two.example.com
+              remoteJwks:
+                cacheDuration: 300s
+                httpUri:
+                  cluster: "127_0_0_1_443"
+                  timeout: 5s
+                  uri: https://127.0.0.1/jwt/public-key/jwks.json
+          requirementMap:
+            requirement-0:
+              providerName: example
+            requirement-1:
+              providerName: example2
+          rules:
+          - requirementName: requirement-0
+          - requirementName: requirement-1
+    - match:
+        path: foo/baz
+      route:
+        cluster: second-route
+      typedPerFilterConfig:
+        envoy.filters.http.jwt_authn:
+          '@type': type.googleapis.com/envoy.extensions.filters.http.jwt_authn.v3.JwtAuthentication
```

Comment:
per_filter_config type in route is [PerRouteConfig](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/jwt_authn/v3/config.proto#envoy-v3-api-msg-extensions-filters-http-jwt-authn-v3-perrouteconfig) which only contains reference to the filter configuration or disabled.